v2/tools/generator/internal/conversions/property_conversions.go

```diff
@@ -101,60 +103,62 @@ func init() {
 //
 // The method works by considering the conversion requested by sourceEndpoint & destinationEndpoint,
 // with recursive calls breaking the conversion down into multiple steps that are then combined.
-//
```

Comment:
It might be possible to make this comment work w/ the code-block style formatting as well, see https://go.dev/play/p/AGebhZXwX7-, which passes `gofmt` so I think should pass the linter too?